content/swiftui/concepts/color/color.md

```diff
@@ -0,0 +1,121 @@
+---
+Title: 'Color'
+Description: 'Creates a color instance.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**`Color`** creates a color instance. `Color`s can be used as arguments, for example, to fill backgrounds and shapes or to apply color to text. They can also be used directly as [views](https://www.codecademy.com/resources/docs/swiftui/views), creating a rectangle area filled with their color, that adapts to fill the space available.
+
+## Syntax
+
+`Color` instances can be declared using dot notation or bracket notation:
+
+```pseudo
+
+let someColor = Color.color property
+
+let someColor = Color(.color property or variable and/or color component values)
+
+```
```

Comment:
It would be best to have one h2 go over the stuct and another, the property (currently lines 29-37)

Under the h2 explaining struct Color, we can have RGB, HSL, and Grayscale as h3s